client/bindings/python/CHANGELOG.md

```diff
@@ -19,6 +19,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## Unreleased - 2023-MM-DD
```

Comment:
Add to RC3 since it's not yet released

Reply:
Guess I can do that to the other branches too 👍 So all my "pythonic" changes. Will do